rotkehlchen/externalapis/beaconchain/service.py

```diff
@@ -360,3 +367,79 @@ def get_eth1_address_validators(self, address: ChecksumEvmAddress) -> list[Valid
                 f'Beaconcha.in eth1 response processing error. Missing key entry {e!s}',
             ) from e
         return validators
+
+    def query_validator_daily_stats(
+            self,
+            validator_index: int,
+            last_known_timestamp: Timestamp,
+            exit_ts: Timestamp | None,
+    ) -> list[ValidatorDailyStats]:
+        """
+        May raise:
+        - RemoteError if we can't query beaconcha.in or if the data is not in the expected format
+        """
+        log.debug(f'Querying daily stats for {validator_index=}')
```

Comment:
Both these logs are not needed. They end up querying beaconchain right? That's loged at `_query()`.